README.md

```diff
@@ -1,75 +1,71 @@
-# Nuxt 3 Minimal Starter
+[![Netlify Status](https://api.netlify.com/api/v1/badges/716a95d1-9f46-4ade-a547-c55d11b3bcb4/deploy-status)](https://app.netlify.com/sites/pantherworld/deploys)
 
-Look at the [Nuxt 3 documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
-
-## Setup
+# Welcome to p4nth3r.world
 
-Make sure to install the dependencies:
+## The game doesn't stop when the stream ends
 
-```bash
-# npm
-npm install
-
-# pnpm
-pnpm install
+p4nth3rworld is a multiplayer text-based game powered by a live coding stream at
+[twitch.tv/whitep4nth3r](twitch.tv/whitep4nth3r).
```

Comment:
The link takes me too _https://github.com/whitep4nth3r/pantherworld/blob/2ca5648a622bf4f41740ef9a1e7e0a7ea00bb1e6/twitch.tv/whitep4nth3r_

Im guessing its due to there being no https:// 

[twitch.tv/whitep4nth3r](https://twitch.tv/whitep4nth3r)